backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -530,4 +475,23 @@
       end
     end
   end
+
+  context 'with only read permissions' do
+    stub_authorization!(permission_sets: [Spree::PermissionSets::OrderDisplay])
+
+    it "should not display forbidden links" do
```

Comment:
For the moment we've explicitly decided as a team that our style guide is limited to [this](https://github.com/solidusio/solidus/blob/15b1fe42698a6d9c501201a067e10d31dfefebf8/CONTRIBUTING.md#follow-the-coding-conventions).  If we want to change that it should be a separate discussion.